static/manifests/calico/DaemonSet/calico-node.yaml

```diff
@@ -171,6 +171,11 @@ spec:
                 configMapKeyRef:
                   name: calico-config
                   key: veth_mtu
+            # Enable the Prometheus metrics server in Felix
+            {{- if .PrometheusMetricsEnabled }}
+            - name: FELIX_PROMETHEUSMETRICSENABLED
+              value: "true"
+            {{- end }}
```

Comment:
If metrics is enabled, should we also put in the standard scrape labels?

Reply:
Probably yes, I could not cover all implementation details